### ISSUE TITLE
Add option to move the [x] to the other side of in game windows.

### ISSUE THREAD

**Issue by PFCKrutonium** [OpenRCT2/OpenRCT2#2468]
For OS X and Various Linux Desktops. Helps keep everything consistent with the OS.

**Comment by IntelOrca** [comment]
* OpenTTD did not do this... and they actually have the X on the other side.
* We can't do this until we re-write the window system anyway because all the widget placement is hard coded.

**Comment by Gymnasiast** [comment]
What OpenTTD did is not really relevant as far as I'm concerned. I think we should implement this eventually, though it's a low priority thing to me.

**Comment by vzybilly** [comment]
For the future, on ubuntu linux, I had a tweak program where I could move
all the elements of the title bar around. I could place them into the left,
center, right and the elements I had to choose from was the Icon, Menu,
Title, minimize, maximize, close. I could even remove all of them which
made for an awkward experience but I could. This might be how we would want
to set this feature up...

side note, it was an old version of ubuntu, while it was still using gnome
and before unity replaced it...

**Comment by IntelOrca** [comment]
@vzybilly this feature is for **in-game** windows.

**Comment by vzybilly** [comment]
I know, I was using a real life example to show how it could be done in
game. it's still a good bit away before we can get coding on it but having
a plan is still good...
